rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2433,6 +2435,35 @@ void voteFederationChange_addFederatorPublicKey_returnsExpectedPendingFederation
 
         }
 
+        @Test
+        void voteFederationChange_whenAddingInvalidFederatorPublicKey_returnsGenericErrorResponseCode() {
+
+            // Arrange
+
+            Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.FIRST_AUTHORIZED.getRskAddress());
+            Transaction tx2 = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.SECOND_AUTHORIZED.getRskAddress());
+
+            ABICallSpec createFederationAbiCallSpec = new ABICallSpec(FederationChangeFunction.CREATE.getKey(), new byte[][]{});
+
+            // Voting with  m of n authorizers to create the pending federation
+            federationSupport.voteFederationChange(tx, createFederationAbiCallSpec, signatureCache, bridgeEventLogger);
+            federationSupport.voteFederationChange(tx2, createFederationAbiCallSpec, signatureCache, bridgeEventLogger);
+
+            byte[] invalidPublicKey = ByteUtil.bigIntegerToBytes(BigInteger.valueOf(123_456_789));
```

Comment:
What makes that magic number special for an invalid public key? Isn't there a different way to create an invalid public key abstracting the use of this literal number?

Reply:
Agree, maybe simply creating an array with length != 32 bytes should be enough to make the key invalid

Reply:
Updated.